pymatgen/io/vasp/tests/test_outputs.py

```diff
@@ -1420,12 +1420,18 @@ def test_get_band_structure(self):
 
 
 class OszicarTest(PymatgenTest):
+    @classmethod
+    def setUpClass(cls):
+        filepath = f"{cls.TEST_FILES_DIR}/OSZICAR"
+        cls.oszicar_mag = Oszicar(filepath)
```

Comment:
We have two `OSZICAR` test files that we could loop over to make maximum use of them. The other is at `test_files/static_silicon/OSZICAR`.

Reply:
Added! 😃

Reply:
Thanks! 👍